metasv/generate_final_vcf.py

```diff
@@ -95,7 +95,7 @@ def convert_metasv_bed_to_vcf(bedfile=None, vcf_out=None, vcf_template_file=vcf_
                 is_pass = (int(interval.fields[8]) != -1) and (svlen == 0 or svlen >= 100)
             sv_id = "."
             ref = "."
-            alt = [vcf.model._SV(sv_type)]
```

Comment:
Looks like the merge did not go well enough. Please change the line back to
original remove line and it should be fine.

On Fri, Sep 4, 2015 at 1:28 PM, msahraeian <notifications@github.com> wrote:

> In metasv/generate_final_vcf.py
> <https://github.com/bioinform/metasv/pull/61#discussion_r38790903>:
>
> > @@ -95,7 +95,7 @@ def convert_metasv_bed_to_vcf(bedfile=None, vcf_out=None, vcf_template_file=vcf_
> >                  is_pass = (int(interval.fields[8]) != -1) and (svlen == 0 or svlen >= 100)
> >              sv_id = "."
> >              ref = "."
> > -            alt = [vcf.model._SV(sv_type)]
>
> I think I didn't made that change. It merged from upstream.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bioinform/metasv/pull/61/files#r38790903>.
>